src/renderer/modules/components/Notice.tsx

```diff
@@ -9,20 +9,19 @@ const NoticeTypes = {
 
 interface NoticeProps {
   children: React.ReactNode;
-  messageType: (typeof NoticeTypes)[keyof typeof NoticeTypes];
+  messageType: (typeof Types)[keyof typeof Types];
   textColor?: string;
   textVariant?: string;
   className?: string;
 }
 
 export type NoticeType = React.ComponentType<NoticeProps> & {
-  NoticeTypes: typeof NoticeTypes;
+  Types: typeof Types;
```

Comment:
This would normally be a breaking change but I think it's ok here since we haven't announced or released the change yet

Reply:
`Notice` comp hasn't been released yet so I think it's ok.